src/main/kotlin/graphics/scenery/volumes/AminoAcidBonds.kt

```diff
@@ -0,0 +1,2 @@
+package graphics.scenery.volumes
```

Comment:
This should not be here at all :-D And `graphics.scenery.volumes` is also the totally wrong package for it.

Reply:
Oh, that wasn't on purpose. :D Good thing you saw it!